server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java

```diff
@@ -74,6 +76,12 @@ public IExchange createExchange(String paramString) //(Bitstamp is in built-in e
                 }
                 return new ItBitExchange(userId, accountId, clientKey, clientSecret, preferredFiatCurrency);
             }
+            else if("hitbtc".equalsIgnoreCase(prefix)) {
```

Comment:
Same as previous comment - change to `} else if("hitbtc".equalsIgnoreCase(prefix)) {`